modules/shared/src/main/scala/org/tessellation/security/signature/Signed.scala

```diff
@@ -106,4 +108,15 @@ object Signed {
         .hashF
         .map(hash => ProofsHash(hash.coerce))
   }
+
+  final class SignedOrdering[A: Ordering] extends Ordering[Signed[A]] {
```

Comment:
Isn't the redefined order for `Signed` enough to derive ordering from it? Won't it give the same result as this definition?